pkg/types/gcp/platform.go

```diff
@@ -30,11 +30,7 @@ type Platform struct {
 	// +optional
 	ComputeSubnet string `json:"computeSubnet,omitempty"`
 
-	// Licenses is a list of licenses to apply to the compute images
-	// The value should a list of strings (https URLs only) representing the license keys.
-	// When set, this will cause the installer to copy the image into user's project.
-	// This option is incompatible with any mechanism that makes use of pre-built images
-	// such as the current env OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE
+	// Licenses support has been removed and must be empty.
```

Comment:
This is a breaking change that we cannot make. We can deprecate the field, but we need to continue to support it.

Reply:
Compromise: we can detect the case where the only thing listed in the licenses is the nested virt, and silently ignore it because we do that by default.

Reply:
Updated now with that compromise!  cc https://github.com/amorenoz from https://github.com/openshift/installer/pull/3430

Reply:
If we had had that restriction to the nested virt license initially, this would be great. Unfortunately, we cannot add a restriction now.

Reply:
I totally understand the PoV that this API was exposed and hence we need to support it ~forever.  If you feel strongly about it, feel free to close this PR.

But again I'm pretty sure no actual customers or CI systems are using anything except nested virt.